handbook/people.md

```diff
@@ -267,6 +267,13 @@ At the end of the quarter, and at key points throughout the quarter (every 3 wee
 - OKRs they are finalized 1 week from when topdown OKRs were initially reviewed.
 - Finalized OKRs are shared company-wide and at "all hands" and, at least to some degree, become public.
 
+## Workversaries
```

Comment:
Welp, none of the spellings are in spell check. I've been spelling it `Workaversaries`, you're spelling it as `Workversaries`.  Both seem pretty reasonable to me.  And yet, [Google thinks it is `Workiversaries`](https://www.google.com/search?q=workiversary)

So I guess we'd better go with that: `Workiversaries`

Reply:
Today I learned, lol. Sounds good to me.